banish/models.py

```diff
@@ -66,14 +69,52 @@ def is_current(self):
 
     class Meta:
         permissions = (("can_ban_user", "Can Ban User"),)
-        verbose_name = "Banishments"
+        verbose_name = "Banishment"
         verbose_name_plural = "Banishments"
         db_table = 'banishments'
 
 
+class Whitelist(models.Model):
+    id = models.AutoField(primary_key=True)
```

Comment:
Thanks for pointing that out. It looks like that's been the case since 1.3 so I'm inclined to remove it from Whitelist and Banishment classes.